internal/aws/cluster/apigateway.go

```diff
@@ -76,9 +78,10 @@ func (a *ApiGateway) Delete() error {
 	return apigateway.DeleteRestApiGateway(a.ResourceName())
 }
 
-func (a *ApiGateway) Create(tags cluster.Tags, privateSubnet bool) error {
+func (a *ApiGateway) Create(tags cluster.Tags) error {
 	vpcId := ""
-	if privateSubnet {
+	if a.ClusterSettings.PrivateSubnet {
+		// TODO: Save it in settings. Cluster update will need to validate that settings have the information and fetch/update one-time in case it's missing, for back compatibility
```

Comment:
By IT I mean vpcId - can be fetched one time on import, saved in DB and then reused to save the call. 
Forst update will not have it, and therefore will need to fix it